forge/src/main/java/whocraft/tardis_refined/common/data/ProviderLootTable.java

```diff
@@ -20,31 +18,22 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Map;
-import java.util.function.BiConsumer;
-import java.util.function.Consumer;
-import java.util.function.Supplier;
+import java.util.Set;
+import java.util.stream.Stream;
 
 public class ProviderLootTable extends LootTableProvider {
 
-    public ProviderLootTable(DataGenerator dataGenerator) {
-        super(dataGenerator);
+    public ProviderLootTable(PackOutput arg) {
+        super(arg, Set.of(), List.of(new LootTableProvider.SubProviderEntry(ModBlockLoot::new, LootContextParamSets.BLOCK), new LootTableProvider.SubProviderEntry(ModEntityLoot::new, LootContextParamSets.ENTITY)));
```

Comment:
Isn't the Set.of() meant to be a set of modids?

Reply:
I don't think so, it is a list of required tables apparently.